.eslintrc

```diff
@@ -14,6 +14,7 @@
     "react/jsx-props-no-spreading": [0],
     "import/no-useless-path-segments": [0, {
       "noUselessIndex": false
-    }]
+    }],
+    "jsx-a11y/label-has-associated-control": "off"
```

Comment:
Could you try this: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md#case-my-label-and-input-components-are-custom-components